### PR TITLE
Initial UI

### PR DESCRIPTION
**Pull request by andresgutgon**
## WAT

Adding initial layout and UI to Katuma

### CODE REVIEW

File: src/helpers/api/apiAjax.js [hunk 1]
```diff
@@ -15,14 +15,12 @@ class ApiAjax {
         const url = formatUrl(path);
 
         req.onload = () => {
-          if (req.status === 500) {
-            reject(req.response);
-            return;
+          if (req.status !== 200) {
```
Comment (enricostano) [inline src/helpers/api/apiAjax.js]: we should take in account more successfull status codes here, like `2**`

File: src/redux/create.js [hunk 2]
```diff
@@ -2,27 +2,29 @@ import { createStore as _createStore, applyMiddleware, compose } from 'redux';
 import createMiddleware from './middleware/clientMiddleware';
 import { syncHistory } from 'react-router-redux';
 
+/**
+ * Check if we can use devtools.
+ *
+ * @return {Boolean}
+ */
+function useDevetools() {
```
Comment (enricostano) [inline src/redux/create.js]: `s/useDevetools/useDevtools/`

File: src/redux/create.js [hunk 3]
```diff
@@ -2,27 +2,29 @@ import { createStore as _createStore, applyMiddleware, compose } from 'redux';
...
-  } else {
-    finalCreateStore = applyMiddleware(...middleware)(_createStore);
-  }
-
   const reducer = require('./modules/reducer');
-  const store = finalCreateStore(reducer, data);
+
+  const store = _createStore(reducer, data, compose(
+    applyMiddleware(...middleware),
+    useDevetools() && window.devToolsExtension ? window.devToolsExtension() : (f) => f
```
Comment (enricostano) [inline src/redux/create.js]: `s/useDevetools/useDevtools/`

**Review by enricostano — COMMENTED** [review]
:muscle: :clap:

File: src/helpers/api/apiAjax.js (hunk 1 above)
Comment (andresgutgon) [reply src/helpers/api/apiAjax.js]: Yes, we need to replace this dummy Ajax implementation by `Zepto` or `fetch-please`. I checked `fetch-please` code and I liked. It has `abort` method which is cool when you want to stop a request in progress.

**Comment by andresgutgon** [comment]
Is this crap ok?
image

File: src/helpers/api/apiAjax.js (hunk 1 above)
Comment (andresgutgon) [reply src/helpers/api/apiAjax.js]: This is super bad 😭 

image

Comment (andresgutgon) [reply src/helpers/api/apiAjax.js]: Lo he arreglado un poco (creo)
https://github.com/coopdevs/katuma-web/pull/15/commits/6f82c196ea8ab256b830e451e907a7d7e0e98cc0

**Comment by sauloperez** [comment]
update your bundler @andresgutgon

**Comment by andresgutgon** [comment]
~~ke dise?~~ ah, ok

**Pull request merged**